recipe/meta.yaml

```diff
@@ -64,24 +65,24 @@ outputs:
         - {{ stdlib("c") }}
         - {{ compiler("cxx") }}
         - python >=3
+        - create-forwarder-dll       # [win]
       host:
       run_constrained:
         - openmp {{ version }}|{{ version }}.*
         - intel-openmp <0.0a0
     test:
       requires:
-        # ensure we don't pull in too-new clang (when on maintenance branch),
-        # which breaks assumption about symlink below
-        - clangxx <={{ version }}
+        - clang-{{ clang_test_version }}
         - {{ compiler('cxx') }}
+        - ld64                      # [osx]
       files:
         - omp_hello.c
       commands:
         # headers
+        - "@echo on"                                      # [win]
         - test -f $PREFIX/include/omp.h                   # [unix]
         - if not exist %LIBRARY_INC%\omp.h exit 1         # [win]
         # clang-specific one (symlink on unix; copies on win, see install_pkg.bat)
-        - test -f $PREFIX/lib/clang/{{ clang_major }}/include/omp.h         # [unix]
```

Comment:
Why did you delete this test? It ensures that the header is where clang expects it?